app/backend/src/couchers/jobs/handlers.py

```diff
@@ -240,13 +240,13 @@ def process_send_reference_reminders(payload):
 
     # Keep this in chronological order!
     reference_reminder_schedule = [
-        # (number, days after stay, text for how long they have left to write the ref)
-        # 6 pm ish two days after stay
-        (1, timedelta(days=2) - timedelta(hours=6), "12 days"),
+        # (number, days before you end of write reference period, text for how long they have left to write the ref)
```

Comment:
spelling error + that it is easier to state how long after the stay ended that the reminder should be sent. It is a bit awkward to have it relative the last time to write reference.

Reply:
Will fix the spelling of course. Did you mean I should switch it back the other way?

I think it's better to have it relative to time left to write the reference... after all, it's a reminder to write a reference before time runs out.

Reply:
Yes but the comments "10 am" etc is really hard to judge if they are correct when we don't know the reference expiration time. And they are easy to forget to update should we change the reference expiration time.

Reply:
I tried to clarify